src/components/NewLexemeForm.vue

```diff
@@ -41,8 +40,11 @@ const language = computed( {
 	get(): SearchedItemOption | null {
 		return store.state.language;
 	},
-	set( newLanguage: SearchedItemOption | null ): void {
-		store.commit( SET_LANGUAGE, newLanguage );
+	async set( newLanguage: SearchedItemOption | null ): Promise<void> {
+		await store.dispatch( HANDLE_LANGUAGE_CHANGE, newLanguage );
+		if ( newLanguage ) {
+			store.commit( CLEAR_PER_FIELD_ERRORS, 'languageErrors' );
```

Comment:
Hm, should this really wait until we’re done looking up the language code of the language item? (But can be switched around later.)

Reply:
Yeah, let's think about this later